bucket/ppsspp-dev.json

```
@@ -0,0 +1,53 @@
+{
+    "homepage": "https://www.ppsspp.org",
+    "description": "Sony PSP emulator (Development}",
+    "version": "1.12.3-905",
+    "license": "GPL 2.0 or later",
+    "architecture": {
+        "64bit": {
+            "url": "https://buildbot.orphis.net/ppsspp/index.php?m=dl&rev=v1.12.3-905-g4c095ce06&platform=windows-amd64",
+            "hash": "9c7500ac24b12e2b6f1f2443ff5929464348215033ab2e5c7606d64cb64a19e3"
+        },
+        "32bit": {
+            "url": "https://buildbot.orphis.net/ppsspp/index.php?m=dl&rev=v1.12.3-905-g4c095ce06&platform=windows-x86",
+            "hash": "2313404076a8b60df4b76df9086efdf26297e0ffe5dcf1f37368d888a21b4cd3"
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\")) {",
+        "   New-item \"$persist_dir\" -ItemType Directory | Out-Null",
+        "   New-item \"$persist_dir\\memstick\" -ItemType Directory | Out-Null",
+        "}"
+    ],
+    "bin": "PPSSPPWindows64.exe",
+    "shortcuts": [
+        [
+            "PPSSPPWindows.exe",
+            "PPSSPP (dev) (32 bit),"
+        ],
+        [
+            "PPSSPPWindows64.exe",
+            "PPSSPP (dev)"
+        ]
+    ],
```

In the interest of simplicity, I would only list 64-bit:

```suggestion
    "shortcuts": [
        [
            "PPSSPPWindows64.exe",
            "PPSSPP (development)"
        ]
    ],
```